tests/test_write_preds.py

```diff
@@ -0,0 +1,202 @@
+import csv
+import os
+import shutil
+import tempfile
+import unittest
+import src.tasks.tasks as tasks
+import torch
+from src import evaluate
+import os.path
+from main import evaluate_and_write
+import pandas as pd
+from src.models import MultiTaskModel
+from unittest import mock
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.data import Instance, Token, vocabulary
+from ..allennlp_mods.numeric_field import NumericField
+from allennlp.data.fields import (
+    LabelField,
+    ListField,
+    MetadataField,
+    TextField,
+)
+
+
+def model_forward(task, batch, predict=True):
+    if task.name == "sts-b":
+        return {"n_exs": 4, "preds": [5.0, 4.0]}
+    return {"n_exs": 4, "preds": [0, 1, 1, 1]}
+
+
+class TestWritePreds(unittest.TestCase):
+    def sentence_to_text_field(self, sent, indexers):
+        """ Helper function to map a sequence of tokens into a sequence of
+        AllenNLP Tokens, then wrap in a TextField with the given indexers """
+        return TextField(list(map(Token, sent)), token_indexers=indexers)
+
+    def setUp(self):
```

Comment:
use snake_case, not camelCase

Reply:
setUp is predefined by the unittest library - it doesn't recognize set_up()

Reply:
I'm following testing code style from examples from the unittest documentation 
https://docs.python.org/3/library/unittest.html